DigitalExcellence/dex-frontend
# Feature/518 relocate buttons details page

## Description

Changed the location of the buttons on the details page from the bottom to the top.

## Type of change


## Checklist


## Steps to Test or Reproduce

1. Open project with call to action(s). If there are not projects with call to action(s), create a new project
2. Validate that the call to action buttons (and embed button if you are logged in as administrator) are at the top. (The embed button should be placed under the call to action button(s))

## Link to issue

Closes: #518
